packages/mux-uploader/src/index.ts

```diff
@@ -394,7 +426,19 @@ class MuxUploaderElement extends HTMLElement {
     });
 
     upload.on('success', () => {
-      console.info('Upload complete!');
+      const successMessage = 'Upload complete!';
+
+      if (this.statusMessage) {
+        this.statusMessage.innerHTML = successMessage;
+      }
+
+      // TO-DO: It seems like statusMessage cannot be updated within two different events. (TD).
```

Comment:
This should be fine, though could be a timing issue. One way you could pin that down would be to break out the `srOnlyText.innerHTML` updates into its own function or method.